server/views/pages/timetable.njk

```diff
@@ -8,28 +8,32 @@
 
 {% set backLinkHref = "/" %}
 
-{% macro attendees(enhanced, prisonerCategoryGroupNames, prisonerLocationGroupNames) %}
-  {% if prisonerLocationGroupNames | length %}
-    <span>
-      {{ "Enhanced prisoners" if enhanced -}}
+{% macro attendees(prisonerIncentiveLevelGroupNames, prisonerCategoryGroupNames, prisonerLocationGroupNames) %}
+    {% if prisonerLocationGroupNames | length %}
+      <span>
+        {% if prisonerIncentiveLevelGroupNames | length %}
+          {{ prisonerIncentiveLevelGroupNames | join("<br>") | safe -}}
+        {% endif %}
+        {% if prisonerCategoryGroupNames | length %}
+          {% if prisonerIncentiveLevelGroupNames | length %},<br>{% endif %}
+          {{ prisonerCategoryGroupNames | join(",<br>") | safe }}
+        {% endif %}
+        {{- " in:" if prisonerIncentiveLevelGroupNames | length or prisonerCategoryGroupNames | length }}
+      </span>
+      <ul class="govuk-list govuk-!-margin-bottom-0{{ " govuk-!-margin-top-2" if enhanced }}">
```

Comment:
```suggestion
      <ul class="govuk-list govuk-!-margin-bottom-0{{ " govuk-!-margin-top-2" if prisonerIncentiveLevelGroupNames | length }}">
```
This just adds a bit of margin above the `<ul>` list of `prisonerLocationGroupNames` so it looks less squashed. Needs updating to no longer use the `enhanced` property.